redbot/cogs/trivia/data/lists/geography.yaml

```diff
@@ -0,0 +1,213 @@
+- Yemen
+What South American country shares the bulk of the Southern Cone with its neighbor Chile to the west?:
+- Argentina
+What is the tallest mountain in North America?:
+- Denali
+- Mount McKinley
+- McKinley
+Montevideo is the capital of what South American country?:
+- Uruguay
+What U.S. state shares borders with Louisiana, Arkansas, Oklahoma, and New Mexico?:
+- Texas
+What is the only continent with land in all four hemispheres?:
+- Africa
+What mountain is closest to the Moon?:
+- Mount Chimborazo
+- Chimborazo
+What continent contains the most fresh water?:
+- Antarctica
+What ocean is home to 75% of the Earth's volcanoes?:
+- Pacific 
+- Pacific Ocean
+What is the largest city in the world based on surface area?:
+- Hulunbuir
+What is the only major city located on two continents?:
+- Istanbul
+What is the coldest sea on Earth?:
+- White Sea
+- White
+What is America's largest city by surface area?:
+- Yakutat
+How many tributaries does the Amazon River have?:
+- 1,100
+What is the oldest active volcano on Earth?:
+- Mount Etna
+- Etna
+Of the eight major islands in the Hawaiian Archipelago, what is the only one that is uninhabited?:
+- Kaho'olawe
+- Kahoolawe
+What river runs through Paris?:
+- Seine
+What mountain range stretches from the town of Seesen in northwest Germany to Eisleben in the east?:
+- Harz
+What Hawaiian island is known as "Bird Island"?:
+- Nihoa
+What mountains form part of the conventional boundary between the continents of Europe and Asia?:
+- Ural Mountains
+- Ural
+What is the fastest flowing river in the world?:
+- Amazon
+How many countries are there in Africa?:
+- 54
+What is the most densely populated country on Earth?:
+- Monaco
+What is the capital of Pakistan?:
+- Islamabad
+What is the most populated nation in South America?:
+- Brazil
+What is the most populated country in Africa?:
+- Nigeria
+What Asian country has Kuala Lumpur as its capital?:
+- Malaysia
+What European country is home to Transylvania?:
+- Romania
+What body of water separates Africa and Europe?:
+- Strait of Gibraltar
+- Gibraltar
+Which of the following continents is intersected by the Equator?:
+- South America
+What is the smallest country in South America?:
+- Suriname
+What is the capital of Nigeria?:
+- Abuja
+Which country is NOT part of the Scaninavian Peninsula?:
+- Denmark
+What is the capital of Turkey?:
+- Ankara
+Lake Titicaca sits on the border between what two nations?:
+- Peru and Bolivia
+- Bolivia and Peru
+- Bolivia Peru
+- Peru Bolivia
+What is the largest country in the world in terms of land area?:
+- Russia
+What U.S. state contains the world's largest flat-top mountain?:
+- Colorado
+What is the state capital of Oklahoma?:
+- Oklahoma City
+What is the oldest city in Texas?:
+- Nacogdoches
+What is the flattest U.S. state?:
+- Florida
+What country borders the Caspian Sea, Persian Gulf, and Gulf of Oman?:
+- Iran
+Which U.S. state has the most miles of rivers?:
+- Nebraska
```

Comment:
What are your sources for this? I found conflicting answers.
<https://www.farmprogress.com/husker-harvest-days/no-nebraska-doesnt-have-most-miles-river>